src/Orleans/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "Newtonsoft.Json": "7.0.1"
+    "Newtonsoft.Json": "9.0.1"
```

Comment:
I would split this version change to a separate commit. I was already planning to remove the dependency on Json for the other projects, as it seemed like it was a transitive dependency, so thanks for it.
The reason I'm saying to split it, is that probably removing Json on other projects should also be done in the `master` branch (as we are using NuGet transitive restore there), so separate commits would make conflicts more manageable (since we don't want to miss any other reference upgrades that may happen in this long lived branch). 'master' can just cherrypick this commit